lib/install/validate/validator_test.go

```diff
@@ -392,12 +457,18 @@ func testStorage(v *Validator, input *data.Data, conf *config.VirtualContainerHo
 			assert.Equal(t, test.expectImage, conf.ImageStores[0].String())
 			conf.ImageStores = conf.ImageStores[1:]
 			for key, volume := range conf.VolumeLocations {
-				assert.Equal(t, test.expectVolumes[key], volume.String())
+				if _, ok := test.expectVolumes[key]; !ok {
+					assert.Fail(t, "Could not find volume store that was expected to present", "volume : %s", volume.String())
+				} else {
+					assert.Equal(t, test.expectVolumes[key].String(), volume.String())
+				}
 			}
+			log.Print("COMPLETED TEST CASE")
```

Comment:
did not mean to leave this behind. I will remove this after the results of the current CI run.